generators/client/templates/angular/src/main/webapp/app/layouts/navbar/navbar.component.html.ejs

```diff
@@ -16,7 +16,7 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -%>
-<nav class="navbar navbar-dark navbar-expand-md bg-dark">
+<nav class="navbar navbar-<% if (clientTheme === 'none' || clientThemeVariant !== 'light') { %>dark<% } else { %>light<% } %> navbar-expand-md <% if (clientTheme === 'none') { %>bg-dark<% } else { %>bg-<%= clientThemeVariant %><% } %>">
```

Comment:
do you need `clientThemeVariant !==` check in first if block? Can it be simplified like 2nd if block?

Reply:
No because we need to also check if `clientThemeVariant` is not light

Reply:
IMO, if user has selected a theme, then, we should use that as is. Second if block is doing that by using theme variant to specify navbar background